lib/services/analysisServices/README.md

```diff
@@ -0,0 +1,85 @@
+# Microsoft Azure SDK for Node.js - AnalysisServices
+
+This project provides a Node.js package for accessing the Azure PAS. Right now it supports:
+- **Node.js version: 4.x.x or higher**
+- **API version: 2016-05-16**
+
+## Features
+
+
+## How to Install
+
+```bash
+npm install azure-arm-analysisservices
+```
+
+## How to Use
+
+### Authentication, client creation and listing role assignments as an example
+
+ ```javascript
+ var msRestAzure = require('ms-rest-azure');
+ var analysisServicesManagement = require("azure-arm-analysisservices");
+ 
+ // Interactive Login
+ // It provides a url and code that needs to be copied and pasted in a browser and authenticated over there. If successful, 
+ // the user will get a DeviceTokenCredentials object.
+ msRestAzure.interactiveLogin(function(err, credentials) {
+  var client = new azureARMAnalysisServices.AnalysisServicesManagementClient(credentials, 'your-subscription-id');
+
+  var util = require('util');
+  var resourceGroupName = 'testrg';
+  var serverName = 'testadlsacct';
+  var location = 'eastus2';
+
+  client.servers.list(function(err, result) {
+    if (err) console.log(err);
+    console.log(result);
+  });
+
+  // account object to create
+  var serverToCreate = {
+    sku: {
+        name: "S1",
+        tier: "Standard"
+    },    
+    tags: {
+      testtag1: 'testvalue1',
+      testtag2: 'testvalue2'
+    },
+    name: serverName,
+    location: location,
+    properties: {
+      provisioningState: "Succeeded",
+      serverFullName: "asazure://stabletest.asazure-int.windows.net/upgradevalidation",
+      asAdministrators: {
+        members: [
+          "aztest0@stabletest.ccsctp.net"
+        ]
+      }
+    }
+  };
+
+  client.servers.create(resourceGroupName, serverName, serverToCreate, options, callback) {
```

Comment:
`callback` doesnt seem to be defined here, did you mean `function(err, result) {..}` ?